packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md

```diff
@@ -23,8 +23,8 @@ If you'd like to explore the entire Auth SDK, please see the [Okta AuthJS Source
 You will need the following things for this guide:
 
 - An Okta org - If you don't have an existing org, register for [Okta Developer Edition](https://developer.okta.com/signup/).
-- An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard-oidc), under the "OpenID Connect Wizard" section.
-- At least one User [assigned to the application](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-apps-page-show-application-embed-links).
+- An OpenID Connect application integration. Instructions for creating one can be found on [this page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard-oidc).
```

Comment:
We avoid making links using references such as "this page"

Can you please work the link into the wording some how?

An OpenID Connect application integration. See the [instructions for creating an application integration]().